apps/web/pages/export.tsx

```diff
@@ -0,0 +1,10 @@
+import {PageHeader} from '../components/PageHeader'
+import {PageLayout} from '../layouts/PageLayout'
+
+export default function ExportPage() {
```

Comment:
To implement in a follow-up PR.

Reply:
We should keep the navigation for the current data-explorer page then its replacement is ready?

Reply:
Agreed, would keep the nav for the old page for now so we can keep using it until the new page is ready

Reply:
3ad9a79